packages/vscode-extension/src/commonlib/telemetry.ts

```diff
@@ -106,6 +106,12 @@ export class VSCodeTelemetryReporter extends vscode.Disposable implements Teleme
       );
     }
 
+    if (TelemetryProperty.ErrorStack in properties) {
+      properties[TelemetryProperty.ErrorStack] = anonymizeFilePaths(
```

Comment:
One idea that can be improved later: Can we keep the filename and remove the file path? We have error stack in some error messages but all file paths are redacted. E.g. `some user stack trace at C:/fake_path/fake_file:1:1` can be anonymized to `some user stack trace at fake_file:1:1`. Since all these files are sourced from our extension, it's not related to user file and can help us better debug.